test/e2e/tests/swaps/swaps-notifications.spec.ts

```diff
@@ -105,20 +105,14 @@ describe('Swaps - notifications', function () {
     );
   });
   it('tests a notification for not enough balance', async function () {
-    const lowBalanceGanacheOptions = {
-      accounts: [
-        {
-          secretKey:
-            '0x7C9529A67102755B7E6102D6D950AC5D5863C98713805CEC576B945B15B71EAC',
-          balance: 0,
-        },
-      ],
```

Comment:
anvil doesn't support setting balances like this, so for achieving compatibility, we just use the default anvil mnemonic -- this will make that our fixtures accounts are empty (no balance) to achieve the testing conditions we want